src/wp-includes/cache.php

```diff
@@ -281,6 +294,33 @@ function wp_cache_flush_runtime() {
 	return wp_cache_flush();
 }
 
+/**
+ * Removes all cache items in a group, if the object cache implementation supports it.
+ * Before calling this method, always check for group flushing support using the
+ * `wp_cache_supports_group_flush()` method.
+ *
+ * @since 6.1.0
+ *
+ * @see WP_Object_Cache::flush_group()
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @param string|array $group name(s) of group to remove from cache.
+ * @return bool|array|WP_Error Bool or array of bool if array passed, WP_Error if not supported.
+ */
+function wp_cache_flush_group( $group ) {
+	global $wp_object_cache;
+
+	// if group is an array, loop and call each group in the array
+	if ( is_array( $group ) ) {
+		return array_combine(
+			array_values( $group ),
+			array_map( 'wp_cache_flush_group', array_values( $group ) )
+		);
+	}
```

Comment:
See above, let's stick with a single `$group` and `bool` return value to make this more consistent with the other cache API functions.
```suggestion
 * @param string $group name Group to remove from cache.
 * @return bool True if group removed, false otherwise.
 */
function wp_cache_flush_group( $group ) {
	global $wp_object_cache;
```